app/src/main/java/ru/faimizufarov/simbirtraining/java/presentation/models/NewsCompose.kt

```diff
@@ -0,0 +1,50 @@
+package ru.faimizufarov.simbirtraining.java.presentation.models
+
+import ru.faimizufarov.simbirtraining.java.domain.models.News
+
+data class NewsCompose(
```

Comment:
Самое главное забыл:
```suggestion
@Immutable
data class NewsCompose(
```
https://developer.android.com/develop/ui/compose/performance/stability/fix#annotate-stable
https://stackoverflow.com/questions/68575936/what-do-the-stable-and-immutable-annotations-mean-in-jetpack-compose

Reply:
впервые встречаю такую аннотацию)